src/App.jsx

```diff
@@ -1,35 +1,31 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState, useEffect } from "react";
+import "./App.css";
+import { Nav } from "./Components/Nav";
+import { Header } from "./Components/Header";
+import { Routes, Route } from "react-router-dom";
+import { Home } from "./Components/Home";
+import { Reviews } from "./Components/Reviews";
+import { fetchAllReviews } from "../utils";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [allReviews, setAllReviews] = useState([]);
+
+  useEffect(() => {
```

Comment:
Why is state for reviews this high up in your app? The component using this data should be making the api call.